package.json

```diff
@@ -1,17 +1,26 @@
 {
-	"license": "MIT",
-	"dependencies": {
-		"@typescript-eslint/eslint-plugin": "^5.51.0",
-		"@typescript-eslint/parser": "^5.51.0",
-		"eslint": "^8.35.0",
-		"eslint-config-prettier": "^8.6.0",
-		"eslint-plugin-prettier": "^4.2.1",
-		"prettier": "^2.8.4",
-		"typescript": "^4.9.5",
-		"vite": "^4.1.4",
-		"vitest": "^0.29.2"
-	},
-	"scripts": {
-		"format": "prettier --write . && eslint --quiet --fix --ext .cjs,.ts ."
-	}
+  "author": "Hugging Face",
+  "repository": "github:huggingface/huggingface.js",
+  "scripts": {
+    "docs:update": "pnpm --filter 'doc-internal' start",
+    "changeset:release": "pnpm --recursive build && pnpm docs:update && changeset publish",
+    "changeset:version": "changeset version",
+    "format": "prettier --write .eslintrc.cjs && eslint --quiet --fix --ext .cjs,.ts .eslintrc.cjs"
+  },
+  "dependencies": {
+    "@changesets/changelog-github": "^0.4.8",
+    "@changesets/cli": "^2.26.0",
+    "@typescript-eslint/eslint-plugin": "^5.51.0",
+    "@typescript-eslint/parser": "^5.51.0",
+    "eslint": "^8.35.0",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "prettier": "^2.8.4",
+    "typescript": "^4.9.5",
```

Comment:
Tabs are converted to spaces too :grimacing:

Reply:
I used the GitHub web editor to resolve conflict in this commit. It should respect the existing indentation style but it doesn't 😕